contents/documents/mobile.html

```diff
@@ -104,6 +104,31 @@ <h3>iOS</h3>
 
 @end</code></pre>
 
+<h4>Integrating Into Xcode</h4>
+
+<p>Part of the appeal of using <code>ebiten</code> is that you don't have to have a lot of experience in Xcode in order to build games for iOS.  Once you import the <code>ebitenmobile</code>-generated framework and setup the View Controller within Xcode, the rest of your coding can take place in Golang.  When you need to test a new change, run the <code>ebitenmobile</code> command again and then Build your project in Xcode.</p>
```

Comment:
`<code>ebiten</code>` is not a term in a code. Use `Ebiten`.

Reply:
Fixed